aurielle-auriellep/hw_4/madlib-with-loops/js/madlib-console.js

```diff
@@ -0,0 +1,25 @@
+$(document).ready(function() {
+
+
+    var favoriteStartups = [];
+    $('#create').on('click', function(e) {
+        var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+        var startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+
+        var random1 = Math.floor((Math.random() * startupX.length));
+        var random2 = Math.floor((Math.random() * startupY.length));
+        e.preventDefault();
+        $('#xForY').html(' ');
+        $('#xForY').html(`A startup that is ${startupX[random1]}, but for ${startupY[random2]}`);
+    });
+    $("#save").on('click', function() {
+        favoriteStartups.push($('#xForY').text());
+        console.log(favoriteStartups);
+    });
+    $("#print").on('click', function() {
+        $("#favorites").append('<ul id="favoritesList"></ul>')
+        favoriteStartups.forEach(function(favorite) {
+            $("#favoritesList").append(`<li>${favorite}</li>`);
```

Comment:
While you're working ES6 into your code, look out for opportunities like this forEach to work arrow functions' lexical `this` into your code for more intuitive handling of the context object you're manipulating. I've seen you use arrow functions while showing your code in class, so I encourage you to continue branching out to learn the new operators available to you. 

Poke through this and other resources to gather some ideas for future projects: http://es6-features.org/

You're doing great, Aurielle!